Tests/TextBufferKitTests/PieceTreeTextBufferTests.swift

```diff
@@ -153,8 +153,7 @@ class PieceTreeTextBufferTests: XCTestCase {
     func testLinesContent(_ str: bstr, _ pieceTable: PieceTreeBase)
     {
         XCTAssert (str == pieceTable.getLinesRawContent ())
-        return
```

Comment:
hm... why the rest of this test is disabled? I enabled it and then it fails now hm... 

https://github.com/migueldeicaza/TextBufferKit/pull/4#issuecomment-830980493